bin/all_sky_search/pycbc_fit_sngls_split_binned

```diff
@@ -121,7 +122,7 @@ bank.close()
 
 # Only calculate params if needed
 usedparams = [args.split_param_two, args.split_param_one, args.bin_param]
-extparams = [par for par in ['mchirp', 'eta', 'chi_eff', 'template_duration']
+extparams = [par for par in bank_conversions.conversion_options
```

Comment:
Looking at the bank conversions code, mass1,2 and spin1,2z are in the options list .. I am not sure I get the thinking here.  'extparams' was extra parameters _beyond_ the ones already in the bank, no? 
Maybe it should be `if (par in usedparams and not par in params)` ?

Reply:
extparams is basically a list of parameters which will need calculation, I think what you've said makes a lot more sense than the loop here, I was shoehorning into what had been done before, but your way makes more sense